deb-s3-upload/action.yml

```diff
@@ -42,4 +45,20 @@ runs:
           CODENAME=${{ inputs.bucket-codename }}
         fi
         echo "Using CODENAME: $CODENAME"
-        find deb-packages -name "*.deb" -print0 | xargs -0 deb-s3 upload --origin=${{ env.AWS_BUCKET_NAME }} --codename ${CODENAME} --component ${{ inputs.component-name }} --bucket ${{ env.AWS_BUCKET_NAME }}
+        echo "Inputs : ${{ inputs.artifact-name }}"
+        echo "Inputs : ${{ inputs.build-architecture }}"
+        if [ -z "${{ inputs.architecture }}" ]; then
+          if [[ "${{ inputs.artifact-name }}" == *"arm64"* ]]; then
+            ARCHITECTURE="arm64"
+          elif [[ "${{ inputs.artifact-name }}" == *"amd64"* || "${{ inputs.artifact-name }}" == *"x64"* ]]; then
+            ARCHITECTURE="amd64"
+          elif [[ "${{ inputs.artifact-name }}" == *"armhf"* ]]; then
+            ARCHITECTURE="armhf"
+          else
+            echo "Unknown architecture"
+          fi
+        else
+          ARCHITECTURE=${{ inputs.architecture }}
```

Comment:
Looks like this condition should return Unknown architecture as well. Can we update this?

Reply:
I think unknown and empty have the same impact since they should just fail when passed to the deb3 command